UnleashClient/__init__.py

```diff
@@ -186,14 +196,44 @@ def is_enabled(self,
             except Exception as excep:
                 LOGGER.warning("Returning default value for feature: %s", feature_name)
                 LOGGER.warning("Error checking feature flag: %s", excep)
-
-                if fallback_function:
-                    fallback_value = default_value or fallback_function(feature_name, context)
-                else:
-                    fallback_value = default_value
-
-                return fallback_value
+                return self._get_fallback_value(default_value, fallback_function, feature_name, context)
         else:
             LOGGER.warning("Returning default value for feature: %s", feature_name)
             LOGGER.warning("Attempted to get feature_flag %s, but client wasn't initialized!", feature_name)
-            return default_value
+            return self._get_fallback_value(default_value, fallback_function, feature_name, context)
+
+
+    # pylint: disable=broad-except
+    def select_variant(self,
+                       feature_name: str,
+                       context: dict = {},
+                       default_value: bool = False,
+                       fallback_function: Callable = None) -> dict:
+        """
+        Checks if a feature toggle is enabled.  If so, return variant.
+
+        Notes:
+        * If client hasn't been initialized yet or an error occurs, flat will default to false.
+
+        :param feature_name: Name of the feature
+        :param context: Dictionary with context (e.g. IPs, email) for feature toggle.
+        :param default_value: Allows override of default value.
+        :param fallback_function: Allows users to provide a custom function to set default value.
+        :return: True/False
```

Comment:
It should return a variant type, not True/False

Reply:
Fixed!